package.json

```diff
@@ -27,6 +27,7 @@
     "dotenv": "^7.0.0",
     "email-validator": "^2.0.4",
     "express": "^4.16.4",
+    "express-restricted": "^1.0.2",
```

Comment:
Which genius authored this?

Reply:
"I remember when express-restricted was at version 1.0.2"

Reply:
🙏